docs/source/deploying-volttron/secure-deployment-considerations.rst

```diff
@@ -0,0 +1,82 @@
+.. _Secure-Deployment-Considerations:
+
+=====================================
+Security Considerations of Deployment
+=====================================
+
+Security of computing systems is a complex topic which depends not only on the
+security of each component, but on how software components interact and on the
+environment in which they are running.
+In the subsections here, we will discuss a variety of possible actions which
+may increase the security of a particular deployment, along with their context.
+
+For more examples and discussion, see the `Publications section of the VOLTTRON website <https://volttron.org/publications>`_ where there are a number of Threat Profile reports.
```

Comment:
keep lines down to 120 or less if possible.

Reply:
fixed